lightning/backend/local.go

```diff
@@ -291,6 +295,7 @@ func NewLocalBackend(
 		tcpConcurrency:    rangeConcurrency,
 		batchWriteKVPairs: sendKVPairs,
 		checkpointEnabled: enableCheckpoint,
+		maxOpenFiles:      utils.MaxInt(maxOpenFiles, minOpenFilesCount),
```

Comment:
`minOpenFilesCount` is confusing, maybe xxxLowerBound

Reply:
agree